spellchecker/symspell.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, initial_capacity=16, max_dictionary_edit_distance=2,
         self._distance_algorithm = DistanceAlgorithm.DAMERUAUOSA
         self._max_length = 0
 
-    def create_dictionary_entry(self, key, count):
+    def create_dictionary_entry(self, key, count, canonical_term=None):
```

Comment:
don't forget typehint please

Reply:
Completely forgot about those. Thanks for reminding me.